63B (#3)
openid-foundation-japan/800-63-4: _sp800-63b/sec4_aal.ja.md

_This section is normative._

<!-- 
To satisfy the requirements of a given AAL and be recognized as a subscriber, a claimant **SHALL** be authenticated with a process whose strength is equal to or greater than the requirements at that level. The result of an authentication process is an identifier that **SHALL** be used each time that subscriber authenticates to that RP. The identifier **MAY** be pseudonymous. Subscriber identifiers **SHOULD NOT** be reused for a different subject but **SHOULD** be reused when a previously enrolled subject is re-enrolled by the CSP. Other attributes that identify the subscriber as a unique subject **MAY** also be provided.
-->

> 与えられた AAL の要件を満たし, Subscriber として認識されるためには, Claimant は, そのレベルの要件と同等, またはそれ以上の強度のプロセスで認証されることとする(**SHALL**). Authentication プロセスの結果は, Subscriber がその RP に対して Authentication する, それぞれの回で使用されることになる(**SHALL**) Identifier である. Identifier は Pseudonymous であってもよい(**MAY**). Subscriber Identifier は別の Subject に再利用されないほうがよい(**SHOULD NOT**)が, 以前に登録された Subject が CSP によって再登録された場合に再利用する必要がある(**SHOULD**). Subscriber を一意の Subject として識別するその他の属性も提供されることがある(**MAY**). 

Reviewer: ```suggestion
与えられた AAL の要件を満たし, Subscriber として認識されるためには, Claimant は, そのレベルの要件と同等, またはそれ以上の強度のプロセスで Authenticate されることとする (**SHALL**). Authentication プロセスの結果として Identifier が提示されるが, この Identifier は毎回 Subscriber がその RP に対して Authentication する度に利用されることとなる(**SHALL**). Identifier は Pseudonymous であってもよい(**MAY**). Subscriber Identifier は別の Subject に再利用されるべきではない(**SHOULD NOT**)が, 以前に登録された Subject が CSP によって再登録された場合に再利用するべきである(**SHOULD**). Subscriber を一意の Subject として識別するその他の Attribute も提供されることがある(**MAY**). 
```

<!--
Detailed normative requirements for authenticators and verifiers at each AAL are provided in [Sec. 5](#AAL_SEC5).

See [[SP800-63]](../_sp800-63/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63"} Sec. 5 for details on how to choose the most appropriate AAL.

[[FIPS140]](references.md#ref-FIPS140) requirements are satisfied by FIPS 140-3 or newer revisions.
-->

各 AAL での Authenticator と Verifier の詳細な Normative Requirement は [Sec. 5](#AAL_SEC5) で提供される.

最も適切な AAL を選択する方法の詳細については [[SP800-63]](../_sp800-63/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63"} Sec. 5 を参照.

[[FIPS140]](references.md#ref-FIPS140) 要件は, FIPS 140-3 か, それ以降のリビジョンで満たされる.

<!--
Personal information collected during and subsequent to identity proofing **MAY** be made available to the subscriber by the digital identity service. The release or online availability of any PII or other personal information, whether self-asserted or validated, by federal government agencies requires multi-factor authentication in accordance with [[EO13681]](references.md#ref-EO13681). Therefore, federal government agencies **SHALL** select a minimum of AAL2 when PII or other personal information is made available online.
-->

> Identity Proofing の最中およびその後に収集された Personal Information は, Digital Identity サービスによって Subscriber が利用できるようになってもよい(**MAY**). 連邦政府機関による任意の PII またはその他の Personal Information のリリース, またはオンラインで利用可能になる場合, self-asserted か validated かにかかわらず, [[EO13681]](references.md#ref-EO13681) に従って Multi-Factor Authentication を必要とする. したがって連邦政府機関は, PII またはその他の Personal Information がオンラインで利用可能になる場合には, 最低限 AAL2 を選択することになる(**SHALL**).

Reviewer: ```suggestion
Identity Proofing の最中およびその後に収集された Personal Information は, Digital Identity サービスを介して Subscriber が利用できるようにしてもよい(**MAY**). 連邦政府機関によって任意の PII またはその他の Personal Information が開示されたりオンラインで利用可能になる場合, その情報が Self-asserted か Validated かにかかわらず, [[EO13681]](references.md#ref-EO13681) に従って Multi-Factor Authentication が要求される. したがって連邦政府機関は, PII またはその他の Personal Information をオンラインで利用可能にする場合には, 最低限 AAL2 を選択することになる(**SHALL**).
```

## Authentication Assurance Level 1

<!--
AAL1 provides some assurance that the claimant controls an authenticator bound to the subscriber account. AAL1 requires either single-factor or multi-factor authentication using a wide range of available authentication technologies. Successful authentication requires that the claimant prove possession and control of the authenticator through a secure authentication protocol.
-->

> AAL1 は, Claimant が Subscriber Account に Bind された Authenticator を制御するというある程度の保証を提供する. AAL1 は, 利用可能な幅広い認証技術を使用した Single-Factor または Multi-Factor の Authentication を必要とする. Authentication の成功には, Claimant がセキュアな Authentication プロトコルを介して Authenticator の所有と制御を証明する必要がある.

Reviewer: ```suggestion
AAL1 は, Claimant が Subscriber Account に Bind された Authenticator を制御しているというある程度の保証を提供する. AAL1 は, 利用可能な幅広い Authentication 技術を使用した Single-Factor または Multi-Factor の Authentication を必要とする. Authentication の成功には, Claimant がセキュアな Authentication プロトコルを介して Authenticator の所有と制御を証明する必要がある.
```

### Authenticator and Verifier Requirements {#aal1req}

<!--
Cryptographic authenticators used at AAL1 **SHALL** use approved cryptography. Software-based authenticators that operate within the context of an operating system **MAY**, where applicable, attempt to detect compromise (e.g., by malware) of the user endpoint in which they are running and **SHOULD NOT** complete the operation when such a compromise is detected.
-->

> AAL1 で使用される Cryptographic Authenticator は承認された暗号を使用することになる(**SHALL**). オペレーティングシステムのコンテキスト内で動作する Software-Based Authenticator は, 可能な場合, それらが実行中のユーザエンドポイントで(例: マルウェアによる)侵害の検出を試みてもよい(**MAY**). また, そのような侵害が検出された場合は操作を完了しないほうがよい(**SHOULD NOT**).

Reviewer: ```suggestion
AAL1 で使用される Cryptographic Authenticator は Approved Cryptography を使用することになる(**SHALL**). オペレーティングシステムのコンテキスト内で動作する Software-Based Authenticator は, 可能な場合, それらが実行中のユーザエンドポイントで(例: マルウェアによる)侵害の検出を試みてもよい(**MAY**). また, そのような侵害が検出された場合は操作を完了しないほうがよい(**SHOULD NOT**).
```

<!--
Communication between the claimant and verifier **SHALL** be via an authenticated protected channel to provide confidentiality of the authenticator output and resistance to adversary-in-the-middle (AitM) attacks.

Verifiers operated by or on behalf of federal government agencies at AAL1 **SHALL** be validated to meet the requirements of [[FIPS140]](references.md#ref-FIPS140-2) Level 1.
-->

> Claimant と Verifier の間の通信は, Authenticator の出力の機密性と Adversary-in-the-Middle (AitM) Attack への耐性の提供のため, 認証された保護されたチャネルを介して行われることになる(**SHALL**).

Reviewer: ```suggestion
Claimant と Verifier の間の通信は, Authenticator の出力の Confidentiality (機密性) と Adversary-in-the-Middle (AitM) Attack への耐性の提供のため, Authenticated Protected Channel を介して行われることになる(**SHALL**).
```

連邦政府機関によって, または連邦政府機関に代わって AAL1 で運用されている Verifier は, [[FIPS140]](references.md#ref-FIPS140-2) Level 1 の要件に適合しているか検証されることになる(**SHALL**).

### Reauthentication {#aal1reauth}

<!--
Periodic reauthentication of subscriber sessions **SHALL** be performed as described in [Sec. 7.2](sec7_session.md#sessionreauthn). At AAL1, reauthentication of the subscriber **SHOULD** be repeated at least once per 30 days during an extended usage session, regardless of user activity. The session **SHOULD** be terminated (i.e., logged out) when this time limit is reached.
-->

> Subscriber Session の定期的な Reauthentication は, [Sec. 7.2](sec7_session.md#sessionreauthn) で説明される通りに実行されることになる(**SHALL**). AAL1 では, Subscriber の Reauthentication は, ユーザのアクティビティに関係なく, 延長使用 Session の間は少なくとも30日に1回繰り返される必要がある(**SHOULD**).

Reviewer: ```suggestion
Subscriber Session の定期的な Reauthentication は, [Sec. 7.2](sec7_session.md#sessionreauthn) で説明される通りに実行されることになる(**SHALL**). AAL1 では, Subscriber の Reauthentication は, ユーザのアクティビティに関係なく, 延長使用 Session の間は少なくとも30日に1回繰り返されるべきである(**SHOULD**). Session はタイムリミットに到達した時点で終了 (i.e., ログアウト) されるべきである (**SHOULD**).
```

### Security Controls

<!--
The CSP **SHALL** employ appropriately tailored security controls from the baseline security controls defined in [[SP800-53]](references.md#ref-SP800-53) or equivalent federal (e.g., [[FEDRAMP]](references.md#ref-FEDRAMP)) or industry standard that the organization has determined for the information systems, applications, and online services that these guidelines are used to protect. The CSP **SHALL** ensure that the minimum assurance-related controls for the appropriate systems, or equivalent, are satisfied.
-->

> CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の Federal (例: [[FEDRAMP]](references.md#ref-FEDRAMP)) で定義されたベースラインセキュリティコントロール, またはこれらのガイドラインを使用して保護する対象となる情報システム, アプリケーション, オンラインサービスのために組織が決定した業界標準から, 適切に誂えられたセキュリティコントロールを採用することになる(**SHALL**). CSP は, 適切なシステムまたは同等のシステムに対する最低限の assurance-related controls が満たされていることを保証することになる(**SHALL**).

Reviewer: ```suggestion
CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の連邦 (例: [[FEDRAMP]](references.md#ref-FEDRAMP)) ないし業界標準で定義されたベースラインセキュリティコントロールから適切に調整されたセキュリティコントロールを採用することになる(**SHALL**). 参考とするガイドラインは, 保護する対象となる情報システム, アプリケーション, オンラインサービスのために組織が決定すること. CSP は, 適切なシステムまたは同等のシステムに対する最低限の assurance-related controls が満たされていることを保証することになる(**SHALL**).
```

-->

CSP は, 適用される可能性のある National Archives and Records Administration (NARA) の記録保持スケジュールを含む, 適用される法律, 規制, およびポリシーに従って, それぞれの記録保持ポリシーを遵守することになる(**SHALL**). CSP が必須要件がない場合に記録を保持することを選択した場合, CSP は, 記録を保持する期間を決定するためにプライバシーおよびセキュリティリスクのアセスメントを含むリスク管理プロセスを実施することになる(**SHALL**). また, そのリテンションポリシーを Subscriber へ通知することになる(**SHALL**).

## Authentication Assurance Level 2

<!--
AAL2 provides high confidence that the claimant controls authenticators bound to the subscriber account. Proof of possession and control of two distinct authentication factors is required through secure authentication protocols. Approved cryptographic techniques are required at AAL2 and above.
-->

> AAL2 は, Claimant が Subscriber Account に Bind された Authenticator を制御するという高い確信を提供する. セキュアな Authentication プロトコルを介して, 2つの明確に異なる認証要素の所有と制御の証明が必要である. AAL2 以上では, 承認された暗号技術が必要である. 

Reviewer: ```suggestion
AAL2 は, Claimant が Subscriber Account に Bind された Authenticator を制御しているという高い確信を提供する. セキュアな Authentication プロトコルを介して, 2つの明確に異なる Authentication Factor の所有と制御の証明が必要である. AAL2 以上では, Approved な暗号技術が必要である. 
```

### Permitted Authenticator Types {#aal2types}

<!--
At AAL2, authentication **SHALL** occur by the use of either a multi-factor authenticator or a combination of two single-factor authenticators. A multi-factor authenticator requires two factors to execute a single authentication event, such as a cryptographically secure device with an integrated biometric sensor that is required to activate the device. Authenticator requirements are specified in [Sec. 5](sec5_authenticators.md#AAL_SEC5).
-->

> AAL2 Authentication は, Multi-Factor Authenticator か, 2つの Single-Factor Authenticator の組み合わせのいずれかを使用することで発生することになる(**SHALL**). Multi-Factor Authenticator は単一の Authentication イベントを実行するために, デバイスをアクティブにするために必要となる Biometric センサーが統合された暗号的にセキュアなデバイスのような, 2つの要素を必要とする. Authenticator の要件は [Sec. 5](sec5_authenticators.md#AAL_SEC5) に指定されている.

Reviewer: ```suggestion
AAL2 Authentication は, Multi-Factor Authenticator か, 2つの Single-Factor Authenticator の組み合わせのいずれかを使用することで発生することになる (**SHALL**). Multi-Factor Authenticator は, 単一の Authentication イベントを実行するために2つの要素を必要とする. 例として, デバイスを Activate するために必要となる Biometric センサーが統合された暗号的にセキュアなデバイスなどが挙げられる. Authenticator の要件は [Sec. 5](sec5_authenticators.md#AAL_SEC5) に指定されている.
```

<!--
When a multi-factor authenticator is used, any of the following **MAY** be used:
-->

Multi-Factor Authenticator が使用されるとき, 以下のいずれかが使用されてもよい(**MAY**).

* Multi-Factor Out-of-Band Authenticator ([Sec. 5.1.3.4](sec5_authenticators.md#mfooba))
* Multi-Factor OTP Device ([Sec. 5.1.5](sec5_authenticators.md#multifactorOTP))
* Multi-Factor Cryptographic Software ([Sec. 5.1.8](sec5_authenticators.md#mfcs))
* Multi-Factor Cryptographic Device ([Sec. 5.1.9](sec5_authenticators.md#mfcd))

<!--
When a combination of two single-factor authenticators is used, the combination **SHALL** include a Memorized Secret authenticator ([Sec. 5.1.1](sec5_authenticators.md#memsecret)) and one physical authenticator (i.e., "something you have") from the following list:
-->

> 2つの Single-Factor Authenticator の組み合わせが使用されるとき, その組み合わせは, 以下のリストから Memorized Secret Authenticator ([Sec. 5.1.1](sec5_authenticators.md#memsecret)) と 1つの物理 Authenticator (すなわち, "something you have") を含むことになる(**SHALL**).

Reviewer: ```suggestion
2つの Single-Factor Authenticator の組み合わせが使用されるとき, その組み合わせは, 以下のリストから1つの Memorized Secret Authenticator ([Sec. 5.1.1](sec5_authenticators.md#memsecret)) と 1つの物理 Authenticator (すなわち, "something you have") を含むことになる(**SHALL**).
```

* Look-Up Secret ([Sec. 5.1.2](sec5_authenticators.md#lookupsecrets))
* Out-of-Band Device ([Sec. 5.1.3](sec5_authenticators.md#out-of-band))
* Single-Factor OTP Device ([Sec. 5.1.4](sec5_authenticators.md#singlefactorOTP))
* Single-Factor Cryptographic Software ([Sec. 5.1.6](sec5_authenticators.md#sfcs))
* Single-Factor Cryptographic Device ([Sec. 5.1.7](sec5_authenticators.md#sfcd))

<!--
> Note: When biometric authentication meets the requirements in [Sec. 5.2.3](sec5_authenticators.md#biometric_use), the device has to be authenticated in addition to the biometric match. A biometric characteristic is recognized as a factor, but not recognized as an authenticator by itself. Therefore, when conducting authentication with a biometric characteristic, it is unnecessary to use two authenticators because the associated device serves as "something you have," while the biometric match serves as "something you are."
-->

> > Note: Biometric Authentication が [Sec. 5.2.3](sec5_authenticators.md#biometric_use) の要件を満たすとき,  Biometric の一致に加えて, デバイスが認証されなければならない. Biometric の特徴は1つの要素として認識されるが, それ自体は Authenticator とは認識されない. したがって, Biometric の特徴による Authentication を行う場合, 関連付けられたデバイスは "something you have" として働き, 同時に Biometric の一致は "something you are" として働くため, 2つの Authenticator を使用することは不要である.

Reviewer: ```suggestion
> Note: Biometric Authentication が [Sec. 5.2.3](sec5_authenticators.md#biometric_use) の要件を満たすとき,  Biometric の一致に加えて, デバイスが Authenticate されなければならない. Biometric の特徴は, それ自体は Authenticator とは認識されないものの, 1つの要素として認識される. したがって, Biometric の特徴による Authentication を行う場合, 関連付けられたデバイスは "something you have" として働き, 同時に Biometric の一致は "something you are" として働くため, 2つの Authenticator を使用することは不要である.
```

### Authenticator and Verifier Requirements {#aal2req}

<!--
Cryptographic authenticators used at AAL2 **SHALL** use approved cryptography. Authenticators procured by federal government agencies **SHALL** be validated to meet the requirements of [[FIPS140]](references.md#ref-FIPS140-2) Level 1. Software-based authenticators that operate within the context of an operating system **MAY**, where applicable, attempt to detect compromise (e.g., by malware) of the platform in which they are running. They **SHOULD NOT** complete the operation when such a compromise is detected. At least one authenticator used at AAL2 **SHALL** be replay resistant as described in [Sec. 5.2.8](sec5_authenticators.md#replay). Authentication at AAL2 **SHOULD** demonstrate authentication intent from at least one authenticator as discussed in [Sec. 5.2.9](sec5_authenticators.md#intent).
-->

> AAL2 で使用される Cryptographic Authenticator は承認された暗号を使用することになる(**SHALL**). 連邦政府機関によって調達された Authenticator は [[FIPS140]](references.md#ref-FIPS140-2) Level 1 の要件に適合していることを検証されることになる(**SHALL**). オペレーティングシステムのコンテキスト内で動作する Software-Based Authenticator は, 可能な場合, それらが実行中のプラットフォームで(例: マルウェアによる)侵害の検出を試みてもよい(**MAY**). また, そのような侵害が検出された場合は操作を完了しないほうがよい(**SHOULD NOT**). AAL2 で使用される少なくとも1つの Authenticator は, [Sec. 5.2.8](sec5_authenticators.md#replay) で説明されているように, リプレイ耐性を持つことになる(**SHALL**). AAL2 での Authentication は, [Sec. 5.2.9](sec5_authenticators.md#intent) で説明されているように, 少なくとも1つの Authenticator から Authentication の意思を実演で示す必要がある(**SHOULD**).

Reviewer: ```suggestion
AAL2 で使用される Cryptographic Authenticator は Approved Cryptography を使用することになる(**SHALL**). 連邦政府機関によって調達された Authenticator は [[FIPS140]](references.md#ref-FIPS140-2) Level 1 の要件に適合していることを検証されることになる(**SHALL**). オペレーティングシステムのコンテキスト内で動作する Software-Based Authenticator は, 可能な場合, それらが実行中のプラットフォームで(例: マルウェアによる)侵害の検出を試みてもよい(**MAY**). また, そのような侵害が検出された場合は操作を完了しないべきである (**SHOULD NOT**). AAL2 で使用される少なくとも1つの Authenticator は, [Sec. 5.2.8](sec5_authenticators.md#replay) で説明されているように, リプレイ耐性を持つことになる(**SHALL**). AAL2 での Authentication は, [Sec. 5.2.9](sec5_authenticators.md#intent) で説明されているように, 少なくとも1つの Authenticator から Authentication Intent を実施する必要がある(**SHOULD**).
```

<!--
Communication between the claimant and verifier **SHALL** be via an authenticated protected channel to provide confidentiality of the authenticator output and resistance to AitM attacks.

Verifiers operated by or on behalf of federal government agencies at AAL2 **SHALL** be validated to meet the requirements of [[FIPS140]](references.md#ref-FIPS140-2) Level 1.
-->

> Claimant と Verifier の間の通信は, Authenticator の出力の機密性と AitM Attack への耐性の提供のため, 認証された保護されたチャネルを介して行われることになる(**SHALL**).

Reviewer: ```suggestion
Claimant と Verifier の間の通信は, Authenticator の出力の Confidentiality (機密性) と AitM Attack への耐性の提供のため, Authenticated Protected Channel を介して行われることになる(**SHALL**).
```

連邦政府機関によって, または連邦政府機関に代わって AAL2 で運用されている Verifier は, [[FIPS140]](references.md#ref-FIPS140-2) Level 1 の要件に適合しているか検証されることになる(**SHALL**).

<!--
When a biometric factor is used in authentication at AAL2, the performance requirements stated in [Sec. 5.2.3](sec5_authenticators.md#biometric_use) **SHALL** be met, and the verifier **SHOULD** make a determination that the biometric sensor and subsequent processing meet these requirements.
-->

> AAL2 での Authentication に Bometric Factorが使用される場合, [Sec. 5.2.3](sec5_authenticators.md#biometric_use) で述べられるパフォーマンス要件を満たすことになり(**SHALL**), Verifier は Bometric センサーとその後の Processing がこれらの要件を満たしていることを判断する必要がある(**SHOULD**).

Reviewer: ```suggestion
AAL2 での Authentication に Bimetric Factorが使用される場合, [Sec. 5.2.3](sec5_authenticators.md#biometric_use) で述べられるパフォーマンス要件を満たすことになり(**SHALL**), Verifier は Bometric センサーとその後の Processing がこれらの要件を満たしていることを判断するべきである(**SHOULD**).
```

<!--
OMB Memorandum [[M-22-09]](references.md#ref-M-22-09) requires federal government agencies to offer at least one phishing-resistant authenticator option to public users at AAL2. While phishing resistance as described in [Sec. 5.2.5](sec5_authenticators.md#verifimpers) is not generally required for authentication at AAL2, verifiers **SHOULD** encourage the use of phishing-resistant authenticators at AAL2 whenever practical since phishing is a significant threat vector.
-->

> OMB Memorandum [[M-22-09]](references.md#ref-M-22-09) は, 連邦政府機関に対して AAL2 のパブリック ユーザーに少なくとも 1 つの Phishing 耐性のある Authenticator のオプションを提供することを要求している. [Sec. 5.2.5](sec5_authenticators.md#verifimpers) で説明されている Phishing 耐性は通常 AAL2 での Authentication には必要とされないが,  Phishing は重大な脅威ベ クトルであるため, Verifier は可能な限り AAL2 でのPhishing 耐性のある Authenticator の使用を奨励する必要がある(**SHOULD**).

Reviewer: ```suggestion
OMB Memorandum [[M-22-09]](references.md#ref-M-22-09) は, 連邦政府機関に対して AAL2 のパブリックユーザーに少なくとも1つの Phishing 耐性のある Authenticator のオプションを提供することを要求している. [Sec. 5.2.5](sec5_authenticators.md#verifimpers) で説明されている Phishing 耐性は通常 AAL2 での Authentication には必要とされないが,  Phishing は重大な脅威ベクトルであるため, Verifier は可能な限り AAL2 での Phishing 耐性のある Authenticator の使用を奨励するべきである(**SHOULD**).
```

### Reauthentication {#aal2reauth}

<!--
Periodic reauthentication of subscriber sessions **SHALL** be performed as described in [Sec. 7.2](sec7_session.md#sessionreauthn). At AAL2, authentication of the subscriber **SHALL** be repeated at least once per 12 hours during an extended usage session, regardless of user activity. Reauthentication of the subscriber **SHALL** be repeated following any period of inactivity lasting 30 minutes or longer. The session **SHALL** be terminated (i.e., logged out) when either of these time limits is reached.

Reauthentication of a session that has not yet reached its time limit **MAY** require only a memorized secret or a biometric in conjunction with the still-valid session secret. The verifier **MAY** prompt the user to cause activity just before the inactivity timeout.
-->

Subscriber Session の定期的な Reauthentication は, [Sec. 7.2](sec7_session.md#sessionreauthn) で説明される通りに実行されることになる(**SHALL**). AAL2 では, Subscriber の Authentication は, ユーザのアクティビティに関係なく, 延長使用 Session の間は少なくとも12時間に1回繰り返されることになる(**SHALL**). Subscriber の Reauthentication は, 30分以上続く非アクティブ期間の後に繰り返されることになる(**SHALL**). これらのタイムリミットのいずれかに到達した場合, Session は終了(つまり, ログアウト)されることになる(**SHALL**).

> タイムリミットにまだ到達していない Session の Reauthentication には, Memorized Secret またはまだ有効な Session Secret と組み合わせた Biometric のみが必要とされるとしてもよい(**MAY**). Verifier は, 非アクティブタイムアウトの直前に, アクティビティを行うようにユーザを促してもよい(**MAY**).

Reviewer: ```suggestion
タイムリミットにまだ到達していない Session の Reauthentication には, Memorized Secret またはまだ有効な Session Secret と組み合わせた Biometric のみが必要とされるようにしてもよい(**MAY**). Verifier は, 非アクティブタイムアウトの直前に, アクティビティを行うようにユーザを促してもよい(**MAY**).
```

### Security Controls

<!--
The CSP **SHALL** employ appropriately tailored security controls from the baseline security controls defined in [[SP800-53]](references.md#ref-SP800-53) or equivalent federal (e.g., [[FEDRAMP]](references.md#ref-FEDRAMP)) or industry standard that the organization has determined for the information systems, applications, and online services that these guidelines are used to protect. The CSP **SHALL** ensure that the minimum assurance-related controls for the appropriate systems, or equivalent, are satisfied.
-->

> CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の Federal (例: [[FEDRAMP]](references.md#ref-FEDRAMP)) で定義されたベースラインセキュリティコントロール, またはこれらのガイドラインを使用して保護する対象となる情報システム, アプリケーション, オンラインサービスのために組織が決定した業界標準から, 適切に誂えられたセキュリティコントロールを採用することになる(**SHALL**). CSP は, 適切なシステムまたは同等のシステムに対する最低限の assurance-related controls が満たされていることを保証することになる(**SHALL**).

Reviewer: ```suggestion
CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の連邦 (例: [[FEDRAMP]](references.md#ref-FEDRAMP)) ないし業界標準で定義されたベースラインセキュリティコントロールから適切に調整されたセキュリティコントロールを採用することになる(**SHALL**). 参考とするガイドラインは, 保護する対象となる情報システム, アプリケーション, オンラインサービスのために組織が決定すること. CSP は, 適切なシステムまたは同等のシステムに対する最低限の assurance-related controls が満たされていることを保証することになる(**SHALL**).
```

-->

CSP は, 適用される可能性のある NARA の記録保持スケジュールを含む, 適用される法律, 規制, およびポリシーに従って, それぞれの記録保持ポリシーを遵守することになる(**SHALL**). CSP が必須要件がない場合に記録を保持することを選択した場合, CSP は, 記録を保持する期間を決定するためにプライバシーおよびセキュリティリスクのアセスメントを含むリスク管理プロセスを実施することになる(**SHALL**). また, そのリテンションポリシーを Subscriber へ通知することになる(**SHALL**).

## Authentication Assurance Level 3 {#aal3}

<!--
AAL3 provides very high confidence that the claimant controls authenticators bound to the subscriber account. Authentication at AAL3 is based on proof of possession of a key through a cryptographic protocol. AAL3 authentication **SHALL** use a hardware-based authenticator and an authenticator that provides phishing resistance — the same device **MAY** fulfill both these requirements. In order to authenticate at AAL3, claimants **SHALL** prove possession and control of two distinct authentication factors through secure authentication protocols. Approved cryptographic techniques are required.
-->

> AAL3 は, Claimant が Subscriber Account に Bind された Authenticator を制御するという非常に高い確信を提供する. AAL3 での Authentication は, 暗号プロトコルを介した Key の所有の証明に基づく. ハードウェアベースの Authenticator  とPhishing 耐性のある Authenticator を使用することになり(**SHALL**), 同じデバイスがこれらの両方の要件を満たしてもよい(**MAY**).　AAL3 で Authentication を行うには, Claimant は, セキュアな Authentication プロトコルを介して, 2つのはっきりと異なる認証要素の所有と制御を証明することになる(**SHALL**). 承認された暗号技術が必要である.

Reviewer: ```suggestion
AAL3 は, Claimant が Subscriber Account に Bind された Authenticator を制御しているという非常に高い確信を提供する. AAL3 での Authentication は, 暗号プロトコルを介した Key の所有の証明に基づく. ハードウェアベースの Authenticator  とPhishing 耐性のある Authenticator を使用することになり(**SHALL**), 同じデバイスがこれらの両方の要件を満たしてもよい(**MAY**).　AAL3 で Authentication を行うには, Claimant は, セキュアな Authentication プロトコルを介して, 2つのはっきりと異なる Authentication Factor の所有と制御を証明することになる(**SHALL**). Approved Cryptographic が必要である.
```

### Authenticator and Verifier Requirements {#aal3req}

<!--
Communication between the claimant and verifier **SHALL** be via an authenticated protected channel to provide confidentiality of the authenticator output and resistance to AitM attacks. At least one cryptographic authenticator used at AAL3 **SHALL** be phishing resistant as described in [Sec. 5.2.5](sec5_authenticators.md#verifimpers) and **SHALL** be replay resistant as described in [Sec. 5.2.8](sec5_authenticators.md#replay). All authentication and reauthentication processes at AAL3 **SHALL** demonstrate authentication intent from at least one authenticator as described in [Sec. 5.2.9](sec5_authenticators.md#intent).
-->

> Claimant と Verifier の間の通信は, Authenticator の出力の機密性と AitM Attack への耐性の提供のため, 認証された保護されたチャネルを介して行われることになる(**SHALL**). AAL3で使用される少なくとも1つの Authenticator は, [Sec. 5.2.5](sec5_authenticators.md#verifimpers) で説明されているよ うにPhishing 耐性があり(**SHALL**), [Sec. 5.2.8](sec5_authenticators.md#replay) で説明されているようにリプレイ耐性があることになる(**SHALL**). AAL3 でのすべての Authentication と Reauthentication は, [Sec. 5.2.9](sec5_authenticators.md#intent) で説明されているように, 少なくとも1つの Authenticator から Authentication の意思を実演で示す必要がある(**SHOULD**).

Reviewer: ```suggestion
Claimant と Verifier の間の通信は, Authenticator の出力の Confidentiality (機密性) と AitM Attack への耐性の提供のため, Authenticated Protected Channel を介して行われることになる(**SHALL**). AAL3で使用される少なくとも1つの Authenticator は, [Sec. 5.2.5](sec5_authenticators.md#verifimpers) で説明されているよ うに Phishing 耐性があり(**SHALL**), [Sec. 5.2.8](sec5_authenticators.md#replay) で説明されているようにリプレイ耐性があることになる(**SHALL**). AAL3 でのすべての Authentication と Reauthentication は, [Sec. 5.2.9](sec5_authenticators.md#intent) で説明されているように, 少なくとも1つの Authenticator から Authentication Intent を実施する必要がある(**SHOULD**).
```

-->

Subscriber Session の定期的な Reauthentication は, [Sec. 7.2](sec7_session.md#sessionreauthn) で説明される通りに実行されることになる(**SHALL**). AAL3 では, [Sec. 7.2](sec7_session.md#sessionreauthn)で説明される通り, Subscriber の Authentication は, ユーザのアクティビティに関係なく, 延長使用 Session の間は少なくとも12時間に1回繰り返されることになる(**SHALL**). Subscriber の Reauthentication は, 15分以上続く非アクティブ期間の後に繰り返されることになる(**SHALL**). Reauthentication は両方の Authentication Factor を使用することになる(**SHALL**). これらのタイムリミットのいずれかに到達した場合, Session は終了(つまり, ログアウト)されることになる(**SHALL**). Verifier は, 非アクティブタイムアウトの直前に, アクティビティを行うようにユーザを促してもよい(**MAY**).

### Security Controls
<!--
The CSP **SHALL** employ appropriately tailored security controls from the baseline security controls defined in [[SP800-53]](references.md#ref-SP800-53) or equivalent federal (e.g., [[FEDRAMP]](references.md#ref-FEDRAMP)) or industry standard that the organization has determined for the information systems, applications, and online services that these guidelines are used to protect. The CSP **SHALL** ensure that the minimum assurance-related controls for the appropriate systems, or equivalent, are satisfied.
-->

> CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の Federal (例: [[FEDRAMP]](references.md#ref-FEDRAMP)) で定義されたベースラインセキュリティコントロール, またはこれらのガイドラインを使用して保護する対象となる情報システム, アプリケーション, オンラインサービスのために組織が決定した業界標準から, 適切に誂えられたセキュリティコントロールを採用することになる(**SHALL**). CSP は, 適切なシステムまたは同等のシステムに対する最低限の assurance-related controls が満たされていることを保証することになる(**SHALL**).

Reviewer: ```suggestion
CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の連邦 (例: [[FEDRAMP]](references.md#ref-FEDRAMP)) ないし業界標準で定義されたベースラインセキュリティコントロールから適切に調整されたセキュリティコントロールを採用することになる(**SHALL**). 参考とするガイドラインは, 保護する対象となる情報システム, アプリケーション, オンラインサービスのために組織が決定すること. CSP は, 適切なシステムまたは同等のシステムに対する最低限の assurance-related controls が満たされていることを保証することになる(**SHALL**).
```

### Records Retention Policy {#aal3records}

<!--
The CSP **SHALL** comply with its respective records retention policies in accordance with applicable laws, regulations, and policies, including any NARA records retention schedules that may apply. If the CSP opts to retain records in the absence of any mandatory requirements, the CSP **SHALL** conduct a risk management process, including assessments of privacy and security risks, to determine how long records should be retained and **SHALL** inform the subscriber of that retention policy.
-->

CSP は, 適用される可能性のある NARA の記録保持スケジュールを含む, 適用される法律, 規制, およびポリシーに従って, それぞれの記録保持ポリシーを遵守することになる(**SHALL**). CSP が必須要件がない場合に記録を保持することを選択した場合, CSP は, 記録を保持する期間を決定するためにプライバシーおよびセキュリティリスクのアセスメントを含むリスク管理プロセスを実施することになる(**SHALL**). また, そのリテンションポリシーを Subscriber へ通知することになる(**SHALL**).

## Privacy Requirements {#aal_privacy}

<!--
The CSP **SHALL** employ appropriately tailored privacy controls defined in [[SP800-53]](references.md#ref-SP800-53) or equivalent industry standard.
-->

> CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の業界標準で定義された, 適切に誂えられたプライバシーコントロールを採用することになる(**SHALL**). 

Reviewer: ```suggestion
CSP は, [[SP800-53]](references.md#ref-SP800-53) または同等の業界標準で定義された, 適切に調整されたプライバシーコントロールを採用することになる(**SHALL**). 
```

<!--
If CSPs process attributes for purposes other than identity proofing, authentication, or attribute assertions (collectively "identity service"), related fraud mitigation, or to comply with law or legal process, CSPs **SHALL** implement measures to maintain predictability and manageability commensurate with the privacy risk arising from the additional processing. Measures **MAY** include providing clear notice, obtaining subscriber consent, or enabling selective use or disclosure of attributes. When CSPs use consent measures, CSPs **SHALL NOT** make consent for the additional processing a condition of the identity service.
-->

> CSP が Identity Proofing, Authentication, Attribute Assertion (総称して "Identity Service"), 関連する不正行為の軽減, または法律や法的手続きの遵守以外の目的で Attribute を処理する場合, CSP は 追加の処理から生じるプライバシーのリスクに見合った Predictability と Manageability を維持するための手段を実装することになる(**SHALL**). 手段には, 明確な通知の提供, Subscriber の同意の取得, Attribute の選択的な使用・開示の有効化を含んでもよい(**MAY**). CSP が同意を手段として使用する場合, CSP は追加の処理の同意を Identity Service の条件にすることはない(**SHALL NOT**).

Reviewer: ```suggestion
CSP が Identity Proofing, Authentication, Attribute Assertion (総称して "Identity Service"), 関連する不正行為の軽減, または法律や法的手続きの遵守以外の目的で Attribute を処理する場合, CSP は 追加の処理から生じるプライバシーのリスクに見合った Predictability と Manageability を維持するための手段を実装することになる(**SHALL**). 手段には, 明確な通知の提供, Subscriber の同意の取得, Attribute の選択的な使用・開示の有効化を含んでもよい(**MAY**). CSP が同意を手段として使用する場合, CSP は追加の処理の同意を Identity Service の条件にしてはならない(**SHALL NOT**).
```
